reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -125,6 +131,16 @@ def iterative_refinement(self, wiener_small_number=0.01, count_norm_const=1):
             .reshape(map_shape)
         )
 
+        xyz_voxels = IterativeRefinement.generate_xyz_voxels(n_pix)
+
+        insert_slice_v = np.vectorize(
```

Comment:
This looks promising! Can you make it into a function and write some tests and docs for it? In the tests you can illustrate some small examples so people understand what the signature corresponds to.

Reply:
Glad to hear it! I'd like to but I don't know of a good way to make np.vectorize outputs into classy methods (we want to avoid vectorizing the function repeatedly). I'll look into it further.